docs/source/resources/01c_object_tracking.rst

```diff
@@ -22,4 +22,30 @@ Benchmarks
 
 .. _object-tracking-benchmarks:
 
-Under Construction
\ No newline at end of file
+Model Accuracy
+--------------
+
+The table below shows the performance of our object tracking models using multiple object tracker
+(MOT) metrics from MOT Challenge. Description of these metrics can be found
+`here <https://motchallenge.net/results/MOT16/#metrics>`__.
+
+
++-------------------------------+------------+-------+-------+--------+-------+--------+
+| Model                         | Type       | MOTA  | IDF1  | ID Sw. | FP    | FN     |
++===============================+============+=======+=======+========+=======+========+
+| IoU Tracker (YOLOX)           | (yolox-m)  | 34.1  | 40.9  | 960    | 8997  | 62830  |
++-------------------------------+------------+-------+-------+--------+-------+--------+
+| OpenCV MOSSE Tracker (YOLOX)  | (yolox-m)  | 32    | 37.6  | 2124   | 7695  | 65268  |
```

Comment:
Would it be clearer if the model were `IoU Tracker with YOLOX`, and the "Type" heading be changed to `Object Detector Type`?
Also, there seems to be a merge conflict with the recent push.

Reply:
Updated according to suggestion.

Fixed merge conflict.